code/modules/reagents/chemistry/reagents/food.dm

```diff
@@ -32,10 +32,10 @@
 				if(prob(50))
 					M.adjustBruteLoss(-1)
 				if(H.species.exotic_blood)
-					H.vessel.add_reagent(H.species.exotic_blood, 0.4)
+					H.blood_volume += 0.4 //will need exoic blood handling
 				else
 					if(!(H.species.flags & NO_BLOOD))
-						H.vessel.add_reagent("blood", 0.4)
+						H.blood_volume += 0.4
```

Comment:
I believe this can all just be

```
		if(H.blood_volume < BLOOD_VOLUME_NORMAL)
			H.blood_volume += 0.4
```

I don't think there need to be a species exotic blood check anymore with this system.